app/modules/intelligence/tools/confluence_tools/confluence_client.py

```
@@ -417,5 +419,5 @@ async def get_confluence_client_for_user(
         )
 
     except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)